src/index.html

```diff
@@ -1,14 +1,26 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="page__content">
+    <div class="watch">
+      <div class="watch__center"></div>
+
+      <div class="watch__hand watch__hand--type--hour">
+        <div class="watch__hand-half watch__hand-half--type--hour watch__hand-half--color--blue"></div>
+        <div class="watch__hand-half watch__hand-half--type--hour"></div>
+      </div>
+
+      <div class="watch__hand watch__hand--type--minute">
+        <div class="watch__hand-half watch__hand-half--type--minute watch__hand-half--color--red"></div>
+        <div class="watch__hand-half watch__hand-half--type--minute"></div>
```

Comment:
can we make bem structure clearer instead of these long elements, and modifiers?